pkgs/by-name/bl/blockstream-electrs/package.nix

```diff
@@ -0,0 +1,92 @@
+{
+  bitcoind,
+  electrum,
+  fetchFromGitHub,
+  lib,
+  rocksdb_8_3,
+  rustPlatform,
+  stdenv,
+}:
+
+rustPlatform.buildRustPackage {
+  pname = "blockstream-electrs";
+  version = "0.4.1-unstable-2024-09-30";
+
+  src = fetchFromGitHub {
+    owner = "Blockstream";
+    repo = "electrs";
+    rev = "dca8c5996a4b69e1544a481e1809d0deaf54c771";
+    hash = "sha256-HAgdJZArFsoX28Ijn+VxjAUwLV9wUW/DLZYyjSiB4TQ=";
+  };
+
+  patches = [
+    # Support bitcoind v28's new default blocksxor flag
+    # Upstream PR: <https://github.com/Blockstream/electrs/pull/130>
+    ./bitcoind-v28.patch
+  ];
```

Comment:
The PR got merged, so you can drop the patch now and update to a later commit. After that I can merge :)